src/mountebank.js

```diff
@@ -148,6 +148,54 @@ function create (options) {
         });
     });
 
+    function portStored (saveImpostersFile, saveImpostersFileFlag) {
+        var fs = require('fs');
+        var portStore = [];
+        var ImposterDir = './Repository_Template';
+        if (fs.existsSync(ImposterDir + '/' + saveImpostersFile) === true) {
+            var textFinal = fs.readFileSync(ImposterDir + '/' + saveImpostersFile, 'utf-8');
+            if ((saveImpostersFileFlag) && (textFinal !== '')) {
+                var parseImposter = JSON.parse(textFinal);
+                (parseImposter.imposters).forEach(function (parse) {
+                    portStore.push(parse.port);
+                });
+                if (portStore.length >= 1) {
+                    logger.warn('These ports are already used ' + portStore + ' use unique ports for imposters. Check ' + ImposterDir + '/' + saveImpostersFile + ' for stored imposter collections');
+
+                }
+            }
+        }
+        else {
+
+            if (!fs.existsSync(ImposterDir)) {
+                fs.mkdirSync(ImposterDir);
+            }
+            fs.writeFileSync(ImposterDir + '/' + saveImpostersFile, '');
+        }
+    }
+
+    function saveImposterFile () {
+        if ((options.savefile === true) && (options.savefile !== undefined)) {
```

Comment:
We created branches because yargs not giving default value that we expect (mb.json). The value of options.savefile is set to `true` instead of `mb.json`(which is default value).
I saw same issue could've been accounted for in this comment in the `mb` file above this function `function fixDefaults () `
 ```
//Unfortunately, while yargs defaults correctly when the command is present, I couldn't
// figure out how to make it default when the command is absent, which I need to default to start.
// You can set what yargs considers to be the default command by .command(['start', '*']), but
// that changes the help docs in ways I don't want.
```

If the value defaults correctly your point is absolutely valid.